app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +50,7 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    setTimeout initModal, 2000 if !window.edscportal
+    preferences.ready.done(-> initModal()) if !window.edscportal
 
   class SearchPage
     constructor: ->
```

Comment:
@mightynimble I do see that - as I described above, any time Preferences need to be accessed, a preferences object must be instantiated.  These two are in two different locations due to scope; one cannot see the other.

In order for the application to check a user preference, it _must_ instantiate a preferences object.  If the preference needs to be checked in two different places in the logic, then that means it must be instantiated two different times (unless the scope is such that it can be used twice, which is not the case here).

However, the instantiation of the preferences object is *not* the problem.  It's not what is causing the race condition or any other issue.  It is actually even 'by design'.  

The race condition would still be a threat even if it were a single instantiation of the preferences object.  The problem lies within the jQuery based AJAX call within the preferences object's constructor.  If the code instantiates a preferences object and then immediately checks a preference, it may accidentally get a default preference as the asynchronous AJAX call within the preference's constructor may not have completed execution.

The beauty of this fix is that it allows communication between the preferences object and the calling code such that it can be confirmed to be 'ready' in a graceful fashion.  It is precise.  The initModal routine will not fire until the deferred object within the preferences constructor is resolved - and it is only resolved within the success condition of the AJAX call.

To sum - yes, the preferences object is instantiated more than once, however this is by design and is not causing the issue.